src/test/java/io/appium/java_client/service/local/ServerBuilderTest.java

```diff
@@ -312,4 +319,56 @@ public void checkAbilityToStartServiceWithLogFileUsingShortFlag() {
         service.start();
         assertTrue(testLogFile.exists());
     }
+
+    @Test
+    public void checkAbilityToStartServiceUsingValidBasePathWithMultiplePathParams() {
+        String baseUrl = String.format("http://%s:%d/", BROADCAST_IP_ADDRESS, DEFAULT_APPIUM_PORT);
+        String basePath = "wd/hub";
+        service = new AppiumServiceBuilder().withArgument(BASEPATH, basePath).build();
+        service.start();
+        assertTrue(service.isRunning());
+        assertEquals(baseUrl + basePath + "/",service.getUrl().toString());
```

Comment:
Please format the stuff everywhere, so there are spaces after comma

Reply:
Noted on this, take care in the latest commit.